### DISCUSSION TITLE
Incorrect Rendering of BFF (MFEs) Content in the Main Application

### DISCUSSION THREAD

**Discussion by rogerlpl** [web-infra-dev/modern.js#4699] [Q&A] [answered]
### Problem Description

I am encountering an issue with the incorrect rendering of content in my main application when attempting to display data coming from the BFF of a MFEs. 

### Questions

1. What considerations should I keep in mind when rendering content from an MFE in a main application?
2. Are there potential conflicts between the configuration or dependencies used in the BFF, MFEs, and the main application that could be causing this issue?

### Additional Information

- The component I'm trying to render from the 'MFEs' is shown in the image below

![image](https://github.com/web-infra-dev/modern.js/assets/11010338/5cfe4aed-7e4e-4e17-aac0-f5e231481bb5)

- The function I invoke from the BFF is this:

![image](https://github.com/web-infra-dev/modern.js/assets/11010338/915c2d77-7fa8-4d6f-8ef4-501a314eff49)

- I have verified that the BFF and the MFE work correctly when displaying data as seen in the image below.

![image](https://github.com/web-infra-dev/modern.js/assets/11010338/b5255a29-d252-4329-bf13-5cf88470a410)

- This is how it appears in the main application.

![image](https://github.com/web-infra-dev/modern.js/assets/11010338/c98f295c-ca25-4840-a6bc-dd3b21cc4fa4)

Any help or guidance on how to address this issue would be greatly appreciated. Thank you for your time and consideration!

**Comment by chenjiahan** [comment]
Can you provide a reproduction repository? This can help us dig into this issue.

**Reply by rogerlpl** [reply]
Sure, you can find the reproduction repository at this link: [Reproduction Repository](https://github.com/rogerlpl/modernjs-bff-mfes). It should help you investigate the issue further.

**Reply by yimingjfe** [reply, marked as answer]
Invoking BFF functions in a sub-application is not supported in the current open source version, I plan to support this feature next week.
